homeassistant/components/yolink/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "yolink",
+  "name": "YoLink",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/yolink",
+  "requirements": ["paho-mqtt==1.6.1", "yolink-client==0.0.8"],
```

We shouldn't need to depend on `paho-mqtt` here. We don't use that directly in the integration. It's `yolink-client` that should require the MQTT library.

```suggestion
  "requirements": ["yolink-client==0.0.8"],
```